OrbitCore/StringManager.cpp

```diff
@@ -0,0 +1,25 @@
+#include "StringManager.h"
+
+std::unique_ptr<StringManager> GStringManager;
+
+StringManager::StringManager() {}
+StringManager::~StringManager() {}
+
+void StringManager::Add(uint64_t key, const std::string& str) {
+  if (key_to_string_.count(key) == 0) {
+    key_to_string_.emplace(key, str);
+  }
+}
+
+std::string StringManager::Get(uint64_t key) {
```

Comment:
We should make the string manager thread safe, string/key pairs will most likely be added from the tcp thread and read from the main thread, or any worker thread.

Reply:
Yes, good point, will change accordingly.